core/graph/src/main/java/uk/gov/gchq/gaffer/graph/Graph.java

```diff
@@ -995,6 +1000,11 @@ private void updateGraphHooks(final GraphConfig config) {
                     config.getHooks().add(0, new NamedOperationResolver());
                 }
             }
+            if (!hasFunctionAuthoriserHook && !config.isSkipDefaultSecurityHooks()) {
+                config.getHooks().add(new FunctionAuthoriser.Builder()
+                        .unauthorisedFunctions(Lists.newArrayList(CreateObject.class))
```

Comment:
Perhaps create a default black list and store that in another class, otherwise this list here could get rather big and messy.